dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -1,4 +1,5 @@
 import os
+import pendulum
 from datetime import datetime, timedelta
```

Comment:
@frankhereford i was just noodling on this in my own dag. if we're going to lean into `pendulum`, i was thinking we might as well get away from the `datetime` imports all together.

what do you think about this? swapping in `duration` for `timedelta`.

```python
from pendulum import datetime, duration


default_args = {
    "owner": "airflow",
    "description": "Publishes all signal stuides records from Knack to Socrata",
    "depends_on_past": False,
    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
    "email_on_failure": False,
    "email_on_retry": False,
    "retries": 0,
    "on_failure_callback": task_fail_slack_alert,
}



with DAG(
    dag_id=f"atd_service_bot_issues_to_dts_portal_{DEPLOYMENT_ENVIRONMENT}",
    default_args=default_args,
    schedule_interval="13 7 * * *",
    dagrun_timeout=duration(minutes=60),
    tags=["repo:atd-service-bot", "knack", "github"],
    catchup=False,
) as dag:

```

Reply:
I love the idea of leaning into pendulum exclusively. I'll update these to do so! Thanks for the great idea.